dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/statelisteners/ElasticListenerTest.java

```diff
@@ -209,6 +210,27 @@ public void testDescriptorTypeVersionsSet() throws IOException {
         assertEquals(0, descriptorTypeVersions.size());
     }
 
+    @Test
+    public void testOpenData() throws IOException {
```

Comment:
the code smell suggestion is worthy, less typing and presumably helps reduce code-completion with test methods we don't intend to call from elsewhere

Reply:
Interesting, I hadn't paid close attention and was in the habit of always making them public, which I think was a JUnit 4 requirement that is no longer in JUnit 5.